src/context/WalletProvider/SelectModal.tsx

```diff
@@ -54,6 +54,9 @@ const WalletSelectItem = ({
 
   const handleConnect = useCallback(() => connect(walletType), [connect, walletType])
 
+  const isLedgerEnabled = getConfig().REACT_APP_FEATURE_LEDGER_WALLET
```

Comment:
there's also `useFeatureFlag` for these use cases

Reply:
All the ways to use feature flags! fdb05a09638689ba97ba0900cc134cef99bc428c